crates/ruma/src/lib.rs

```diff
@@ -34,9 +34,9 @@
 //!
 //! These features are only useful if you want to use a method that requires it:
 //!
-//! * `either`
```

Comment:
Leftover?

Reply:
It looks like this feature was removed but not from the docs :shrug: